acceptance-tests/src/test/scala/ch/datascience/graph/acceptancetests/WebhookCreationSpec.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2019 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ch.datascience.graph.acceptancetests
+
+import ch.datascience.generators.CommonGraphGenerators.accessTokens
+import ch.datascience.generators.Generators.Implicits._
+import ch.datascience.graph.acceptancetests.stubs.GitLab._
+import ch.datascience.graph.acceptancetests.tooling.GraphServices
+import ch.datascience.graph.acceptancetests.tooling.ResponseTools._
+import ch.datascience.graph.model.events.EventsGenerators.projectIds
+import ch.datascience.http.client.AccessToken.{OAuthAccessToken, PersonalAccessToken}
+import ch.datascience.webhookservice.project.ProjectVisibility.Public
+import io.circe.literal._
+import org.http4s.Status._
+import org.scalatest.Matchers._
+import org.scalatest.{FeatureSpec, GivenWhenThen}
+
+class WebhookCreationSpec extends FeatureSpec with GivenWhenThen with GraphServices {
+
+  feature("The user can add a webhook to a project with POST webhook-service/projects/:id/webhooks") {
+
+    scenario("There's already Graph Services webhook on the project in GitLab") {
```

Comment:
Isn't this going to be different for public and private projects?

Reply:
No, it's the same for all visibilities.